pkg/release/official/client.go

```diff
@@ -16,7 +16,11 @@ import (
 	"github.com/openshift/ci-tools/pkg/release"
 )
 
-const cincinnatiAddress = "https://api.openshift.com/api/upgrades_info/v1/graph"
+const (
+	cincinnatiAddressProd        = "https://api.openshift.com/api/upgrades_info/v1/graph"
+	cincinnatiAddressStage       = "https://api.stage.openshift.com/api/upgrades_info/v1/graph"
+	cincinnatiAddressIntegration = "https://api.integration.openshift.com/api/upgrades_info/v1/graph"
```

Comment:
You can drop the `/v1` here to catch up with openshift/cincinnati#537 and [OTA-531][1].  But that's largely orthogonal, so it could also be punted out to follow-up work.

[1]: https://issues.redhat.com/browse/OTA-531